src/Gui/PythonWrapper.cpp

```diff
@@ -362,12 +369,8 @@ Py::Object qt_wrapInstance(qttype object,
                            const std::string& wrap)
 {
     PyObject* module = PyImport_ImportModule(shiboken.c_str());
-    if (!module) {
-        std::string error = "Cannot load ";
-        error += shiboken;
-        error += " module";
-        throw Py::Exception(PyExc_ImportError, error);
-    }
+    if (!module)
+        throw Py::Exception(PyExc_ImportError, formatModuleError(shiboken));

```

Comment:
Could you add curly braces here (and below)? This code isn't yet covered by our auto-formatting CI, but it will be eventually, and this will be one less thing we have to fix.

Reply:
Done, force-pushed (this is something I really do not like, so I need discipline to verify changes against coding policy :))

Reply:
If your IDE supports clang-format and is capable of applying it locally (e.g. not to the entire file, but only to selected portions), that is probably your easiest option, while we slowly migrate our codebase to using it.

Reply:
[Vim](https://www.vim.org/) is my IDE, I'll look into clang-format later. Thank you for suggestion.

Reply:
Perfect, the clang-format plugin for vim works on isolated text selections, which is exactly what's needed.

Reply:
[Vim](https://www.vim.org/) is my IDE. I'll look into clang-format later, thank for suggestion. (Now it seems main branch was merged, while rebase was intended, confusing... Should I fix it and force push or are you able to resolve it yourself?)

Reply:
Don't worry about it, I'll clean it up when I merge.